t/ui/18-tests-details.t

```diff
@@ -162,6 +162,16 @@ subtest 'route to latest' => sub {
     $get    = $t->get_ok($baseurl . 'tests/latest?test=kde&machine=32bit')->status_is(200);
     $header = $t->tx->res->dom->at('#info_box .panel-heading a');
     is($header->{href}, '/tests/99937', 'also filter on machine');
+    my $job_groups_links      = $t->tx->res->dom->find('.nav .dropdown a + ul.dropdown-menu li a');
+    my @job_groups_link_texts = $job_groups_links->map('text')->each;
+    my @job_groups_link_hrefs = $job_groups_links->map('attr', 'href')->each;
```

Comment:
can you expand the array into two scalar variables with a good name so that we know better what the values within each array mean? e.g. `my ($job_group_text, $build_text) = …`